lib/folio_client/inventory.rb

```diff
@@ -22,5 +22,21 @@ def fetch_hrid(barcode:)
       result = client.get("/inventory/instances/#{instance_uuid}")
       result.dig("hrid")
     end
+
+    # @param hrid [String] folio instance HRID
+    # @param status_code [String] id for the "cat" instance status code
```

Comment:
Why the naming mismatch between status code and status id?

Reply:
Maybe this param should be called `status_code_id`? There are instance status codes such as "cat" and they have a `name` (e.g. "Cataloged") and an `id`. The `id` is a UUID for that code.

Reply:
Or I could use the same field name as folio, which would be `status_id`. I think that's actually what you're suggesting?

Reply:
the param could be named "status_uuid" or "status_code_uuid" -- making it clear a) it's a uuid and b) it's definitely the id of a record in folio, not the status string itself.    (clear as folio mud?)

Reply:
I don't have a specific suggestion, but by the time I got to `status_id == status_code`, I was confused.